### PR TITLE
Ftv 275 linux build

### PR DESCRIPTION
**Pull request by clusty**
This branch includes precanned linux binaries.
Summary of changes:
*CMake change to mitigate HDF5 errors on Ubuntu.
*Standalone build fix: initially disabled Alembics were not being copied into the app.
*Precanned linux binary and instructions how to build on CentOS

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,7 +51,7 @@ install/**
 External/install/**
 com.unity.formats.alembic/.npmignore
 com.unity.formats.alembic/package.json
-com.unity.formats.alembic/Runtime/Plugins
+#com.unity.formats.alembic/Runtime/Plugins
```
Comment (mfe) [inline .gitignore]: Just remove the line

File: com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicMesh.cs [hunk 2]
```diff
@@ -327,12 +327,12 @@ public override void AbcSyncDataEnd()
                         split.mesh.SetUVs(3, split.velocities.List);
 #endif
                     }
-
```
Comment (mfe) [inline com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicMesh.cs]: Does not feel that this two changes are needed. I would remove them.

File: .gitignore (hunk 1 above)
Comment (clusty) [reply .gitignore]: This is temporary. As soon as I have a valid linux CI, this will be put back.

**Comment by clusty** [comment]
> Minor remarks.
> 
> Can you explain why the CI is not working in the branch ? Will dev be broken too after the merge ?

The CI is failing because of a new validation suite test was added to ensure SemVer compliance: **Thou shalt not add a new supported platform during a patch release** : We are doing exactly that. I think the dev CI will be broken till this package is published: the test looks at the API delta between the last published version and version to be tested. Will confirm.

**Pull request merged**